src/sentry/issues/occurrence_consumer.py

```diff
@@ -319,6 +320,21 @@ def process_occurrence_message(
         txn.set_tag("result", "dropped_feature_disabled")
         return None
 
+    # Rate limit based on fingerprint
```

Comment:
let's use an option for the rollout of the rate limiter and have it be set default to false, and we can turn it on with high limits, and observe no problems, and remove it once we ensure that its rolled out safely.